qa/automation/ScoringPortal.Tests.Scenarios/3.Sign_in_with_GoogleAcc/1. sign_in.feature

```diff
@@ -0,0 +1,36 @@
+Feature: 3.1 Sign in with Google Account
+
+Background:
+	Given user turn on VPN with necessary config
+	And go to https://qa.ddso-spot.quantori.com/
+
+Scenario: 3.1_01 User has Google account
+	Given User has a saved Google account
+	When User press the button "Sign in with Google"
+	And User select necessary Google account 
+	Then the page with Quantori Scoring is opened
+
+Scenario: 3.1_02 User doesn't have a saved Google account
+	Given User has a google account but it isn't saved in the browser
+	When User press the button "Sign in with Google"
+	And User see the "Sign in to your Google account" field
+	And User fills in all necessary information
```

Comment:
Needs some more information, what data can we use, what is necessary and what changes. Try to split this scenario more, because too many And's in When mean that this scenario isn't specific enough.

Reply:
Sorry, I don't understand the comment, especially "what is necessary and what changes" part. Can you please clarify it?

Reply:
I mean, you need to clarify "necessary information" in line 17: what is it, what parts it contains, and what exactly we can put in the fields, probably with examples. And if your "When" has too many "And"'s (more than 0), this means that you can split the scenario or move some of the "And"'s from "When" to "Given".

Reply:
Yeah, but you still need to write down what necessary information is, at least "email" and "password" is enough for clarifying. You need to be precise in steps of the test case.

Reply:
In this case I agree with @dmitri-kandalintsev  - 
1) we should use specific test data in scenarios in order to check it. It means that you need to add concrete emails, passwords  
2) a lot of lines in When step - not great 
3) line "User fills in all necessary information" - too general description and it looks like some steps were missed, e.g. to click on "Next" button